packages/trader/src/Modules/Contract/Components/Digits/digits.tsx

```diff
@@ -77,24 +122,24 @@ const DigitsWrapper = ({
     );
 };
 
-const Digits = React.memo(props => {
-    const [status, setStatus] = React.useState();
-    const [current_tick, setCurrentTick] = React.useState();
-    const [spot, setSpot] = React.useState();
-    const [is_selected_winning, setIsSelectedWinning] = React.useState();
-    const [is_latest, setIsLatest] = React.useState();
-    const [is_won, setIsWon] = React.useState();
-    const [is_lost, setIsLost] = React.useState();
+const Digits = React.memo((props: TDigits) => {
+    const [status, setStatus] = React.useState<string | null>();
+    const [current_tick, setCurrentTick] = React.useState<number | null>();
+    const [spot, setSpot] = React.useState<string>();
+    const [is_selected_winning, setIsSelectedWinning] = React.useState<boolean>();
+    const [is_latest, setIsLatest] = React.useState<boolean>();
+    const [is_won, setIsWon] = React.useState<boolean>();
+    const [is_lost, setIsLost] = React.useState<boolean>();
     const isMounted = useIsMounted();
 
     const { contract_info, digits_array, is_digit_contract, is_trade_page, underlying } = props;
 
-    const onChangeStatus = params => {
+    const onChangeStatus: React.ComponentProps<typeof DigitsWrapper>['onChangeStatus'] = params => {
```

Comment:
Can't we use just `TOnChangeStatus`? Same for `TOnLastDigitSpot`

Reply:
lol yeah same same. Doesnt make a difference